src/imv.c

```diff
@@ -479,6 +483,7 @@ struct imv *imv_create(void)
   imv->need_redraw = true;
   imv->need_rescale = true;
   imv->scaling_mode = SCALING_FULL;
+  imv->custom_start_pan = false;
```

Comment:
`imv` is explicitly zero-initialised, so `0` values don't need to be set. Please take this line out.